lino/modlib/openui5/config/openui5/elems/column/FieldElement.xml

```diff
@@ -7,14 +7,12 @@
         sortProperty="{{c.field.name}}"
         showSortMenuEntry="true"
         {% endif %}
-        {% if c.field.name in ar.actor.tablet_columns %}
-        minScreenWidth="Tablet"
-        {% endif %}
-        {% if c.field.name in ar.actor.mobile_columns %}
-        minScreenWidth="Mobile"
-        {% endif %}
-        {% if c.field.name in ar.actor.popin_columns %}
-        demandPopin="true"
+        {% if c.field in ar.actor.tablet_columns and c.field.name != ar.actor.column_names.split(' ')[0] %}
```

Comment:
I guess you mean "if c.field.name ..." instead of "if c.field ..."

Reply:
No, it doesn't work as you suggest, because popin_columns contains set fields and not set of fields' names.

Reply:
You are right, I see now.